### PR TITLE
Update AutoYaST documentation

### PR DESCRIPTION
**Pull request by imobachgs**

### CODE REVIEW

File: doc/autoyast.md [hunk 1]
```diff
@@ -4,23 +4,66 @@ When using Containers as a Service Platform (CaaSP), AutoYaST profiles for
...
+```xml
+<chroot-scripts config:type="list">
+  <script>
+    <chrooted config:type="boolean">true</chrooted>
+    <filename>configure-ntpd.sh</filename>
+    <interpreter>shell</interpreter>
+    <source><![CDATA[
+#!/bin/sh
+sed '/^server/d' -i /etc/ntp.conf
+for server in server1 server2 server3
```
Comment (kobliha) [inline doc/autoyast.md]: Just a little note: It's not that obvious that server1 server2 server3 should be replaced with `your` NTP servers.

Comment (kobliha) [inline doc/autoyast.md]: Oops, but it's documented

**Review by schubi2 — APPROVED** [review]
LGTM

**Comment by imobachgs** [comment]
Thanks!

**Pull request merged**